### ISSUE TITLE
Support $PAGER environment variable

### ISSUE THREAD

**Issue by neilb** [ggreer/the_silver_searcher#965]
Would be nice if I could set $PAGER and it would auto page, rather than having to use `--pager`

Cheers,
Neil

**Comment by hansfn** [comment]
Yes, in particular since `--pager PROG` already is ignored "if the output is piped to another program".

@forgottenswitch Any reason you closed your PR?

**Comment by forgottenswitch** [comment]
On the technical side, none; however, The Silver Searcher does seem to be
feature-frozen. So, as long as somebody wants anything like this, it is
suggested - I believe - that they just fork the source code. Hence, no reason
to leave the PR open; it consists of trivia, after all.

*Maybe* the feature freeze is caused in part by the rise of the Rust
programming language, as it looks like a "better C".
